### DISCUSSION TITLE
How to set FocusingPosition and AlignAtFocusingPosition at runtime?

### DISCUSSION THREAD

**Discussion by feipinxiang** [LanKuDot/game_modules#44] [Q&A] [answered]
In some cases, ListBank data is dynamic. I want Foucs at center when bank data is less than NumOfBoxes, Focus at bottom when data is greater than NumOfBoxes.

**Comment by LanKuDot** [comment, marked as answer]
Sorry, once the list is initialized. The setting couldn't be changed.
The setting setter is provided for setting up the list which is spawned in runtime.

**Reply by feipinxiang** [reply]
ok. thx
